kasa/smartbulb.py

```diff
@@ -533,3 +533,8 @@ async def save_preset(self, preset: SmartBulbPreset):
         return await self._query_helper(
             self.LIGHT_SERVICE, "set_preferred_state", preset.dict(exclude_none=True)
         )
+
+    @property
+    def max_response_payload_size(self) -> int:
```

Comment:
Although I'm fine with the current naming, maybe `max_device_response_size` would make its purpose clearer?